### ISSUE TITLE
1.20.1 Hotfix 1 Forge release crashing server

### ISSUE THREAD

**Issue by ImKringle** [Minecraft-Transit-Railway/Minecraft-Transit-Railway#743]
**Expected Behaviour**
Server boots

**Actual Behaviour**
Server did *not* boot lol

**Reproduction Steps**
Steps to reproduce the behaviour:

1. Install the latest forge alpha release to the server
2.  Boot server
3. watch it crash

**Crash Report**
https://pastebin.com/b2iksA0s

**Screenshots**
N/A

**Environment**

- Minecraft Version: 1.20.1 
- Installed Mods: This modpack -> https://www.curseforge.com/minecraft/modpacks/minecolonies-dimensional-adventure/files/4707164

**Additional Information**
After installing the pack and removing client side mods the server crashing due to client data from the MTR mod - not sure on the fix just figured id send up the pipe 🖖 

[crash-2023-08-25_06.12.39-fml.txt](https://github.com/jonafanho/Minecraft-Transit-Railway/files/12436192/crash-2023-08-25_06.12.39-fml.txt)

**Comment by kettle-7** [comment]
pretty sure 1.20 still isn't supported on servers yet

**Comment by Jissee** [comment]
Please refer to #723 and #741

**Comment by kettle-7** [comment]
specifically for mtr
mtr doesn't support 1.20 servers

**Comment by kettle-7** [comment]
waiting on [jonafanho/Minecraft-Mappings](https://github.com/jonafanho/Minecraft-Mappings)

**Comment by Kenny-Hui** [comment]
Duplicate of #701
